disperser/apiserver/server.go

```diff
@@ -260,7 +260,7 @@ func (s *DispersalServer) DisperseBlob(ctx context.Context, req *pb.DisperseBlob
 // to track the error again.
```

Comment:
similarly, not necessarily in this PR, but we'd need some clean up in v1 server to remove payments